dbexporter/docker-compose-fem-dbexporter-postgresql.yml

```diff
@@ -7,8 +7,8 @@ services:
   fem_postgresql-exporter:
     image: prometheuscommunity/postgres-exporter:v0.15.0
     restart: unless-stopped
-    container_name: fem_postgresql_exporter
-    hostname: fem_postgresql_exporter
+    container_name: fem-postgresql_exporter
+    hostname: fem-postgresql_exporter
```

Comment:
Since it is part of the hostname, I suppose the other "_" should also be changed. Do you agree?